newdle/templates/joined_email.txt

```diff
@@ -0,0 +1,10 @@
+Hi,
+
+{{ participant }} replied to "{{ title }}".
+
+Selected dates:
+{% for item in answers -%}
+    - {{ item }}
+{% endfor %}
```

Comment:
If the user left a comment when answering the newdle including this might be nice as well...

Reply:
Something like this? @ThiefMaster 

`'comments': participant.comment if participant.comment else 'No comments added'`

```
Selected dates:
Hi,

{{ participant }} replied to "{{ title }}".
{{ comments }}

Selected dates:
{% for item in answers -%}
    - {{ item }}
{% endfor %}
```

Reply:
I would only include something if there is a comment. Something like this (example not taking whitespace into account:

```jinja
{% if comment %}
Comment: {{ comment }}
{% endif %}
```